docs/examples/network-policies/README.md

```diff
@@ -0,0 +1,23 @@
+# Network Policy Example
+
+Kubernetes allows you to restrict the source/destination of traffic to & from your Pods at an IP / port level, by defining NetworkPolicies for your cluster, provided your
+cluster has the [network plugin](https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/network-plugins/) enabled. For more
+information on NetworkPolicies, see the [Kubernetes documentation](https://kubernetes.io/docs/concepts/services-networking/network-policies/).
+
+By defining NetworkPolicies, you can restrict the network entities with which your RabbitmqCluster can communicate, and prevent unrecognised traffic
+from reaching the cluster. It is important to note that once a RabbitmqCluster Pod, or any other Pod for that matter, is the target of any
+NetworkPolicy, it becomes isolated to all traffic except that permitted by a NetworkPolicy.
+
+The first policy in this example, [allow-inter-node-traffic.yaml](./allow-inter-node-traffic.yaml) ensures that the only nodes in the RabbitmqCluster
+send or receive traffic with each other on the ports used for inter-node communication.
+
+The second policy, [allow-operator-traffic.yaml](./allow-operator-traffic.yaml), allows the cluster-operator and the messaging-topology-operator to
```

Comment:
I think only the admin password update logic from cluster operator connects to RMQ via HTTP. Do you think it's worth calling it out?

Reply:
I figured I'd leave it generic so that if we have cause to expand it later it's not going to break anything by being to restrictive. Similarly if we merge the two operators - they both have the label `app.kubernetes.io/component: rabbitmq-operator` so nothing needs to change there.